src/main/service/GeocodingService.ts

```diff
@@ -0,0 +1,40 @@
+import nodeGeocoder, { Entry, Geocoder, Options } from 'node-geocoder';
+import { GeocodingObject } from '../data/GeocodingObject';
+import * as dotenv from 'dotenv';
+
+/**
+ * A Node library for geocoding our restaurant addresses
+ */
+export class GeocodingService {
+  private geocoder: Geocoder;
+
+  constructor() {
+    dotenv.config();
+    const options: Options = {
+      provider: 'locationiq',
+      apiKey: process.env.GEOCODING_KEY,
```

Comment:
Do you have a link for where u signed up for api key? I'm just curious cause gonna look into the usage limit and other stuff

Reply:
Yup, https://locationiq.com/pricing

There's no billing required during signup and the free tier accepts 5000 quests/day

Reply:
Could you add it to `sample.env` and also add the key in the Google Drive folder?